x-pack/plugins/security_solution/scripts/endpoint/common/roles_users/t2_analyst.ts

```diff
@@ -17,14 +17,7 @@ export const getT2Analyst: () => Omit<Role, 'name'> = () => {
         ...noResponseActionsRole.kibana[0],
         feature: {
           ...noResponseActionsRole.kibana[0].feature,
-          siem: [
-            'minimal_all',
-            'actions_log_management_read',
-            'trusted_applications_read',
-            'event_filters_read',
-            'host_isolation_exceptions_read',
-            'blocklist_read',
-          ],
+          siem: ['minimal_all', 'actions_log_management_read'],
```

Comment:
is this change not breaking the `artifacts` test?

Reply:
looks like, it breaks everything 😅 I'll move this modification to a separate PR so this one can be merged

Reply:
140e7ae